bentoml/yatai/yatai_service_impl.py

```diff
@@ -393,7 +393,7 @@ def GetBento(self, request, context=None):
                 logger.info(
                     'Getting latest version %s:%s',
                     request.bento_name,
-                    bento_pb.version,
+                    request.bento_version,
                 )
```

Comment:
I think `bento_pb.version` here is intentional, so that users using 'latest' tag, gets to know the exact version got fetched.

Can we fix the issue by moving L392-397 to the `if bento_pb` block below?

```
     if bento_pb:
            if request.bento_version.lower() == 'latest':
                logger.info(
                    'Getting latest version %s:%s',
                    request.bento_name,
                    bento_pb.version,
                )

        if bento_pb.uri.type == BentoUri.S3:
             ...
```

Reply:
yeah that could work,but it won't log if bento is not found.

Reply:
On L406 it will actually return a proper error message when the bento is not found

Reply:
ok,changing accordingly